b9/src/MethodBuilder.cpp

```diff
@@ -335,11 +335,13 @@ bool MethodBuilder::generateILForBytecode(
   }
 
   switch (instruction.opCode()) {
+    case OpCode::PUSH_FROM_ARG:
     case OpCode::PUSH_FROM_VAR:
       push(builder, loadVarIndex(builder, instruction.immediate()));
       if (nextBytecodeBuilder)
         builder->AddFallThroughBuilder(nextBytecodeBuilder);
       break;
+    case OpCode::POP_INTO_ARG:
```

Comment:
OK, looking at the frontend, the parameters and locals each use a different symbol table. This means that args and params are indexed differently; arg[0] is not the same as local[0]. But here, it looks like arg[0] and local[0] are aliased. Is this a bug? There's the same issue in the interpreter.